package.json

```diff
@@ -45,6 +47,7 @@
   "devDependencies": {
     "browserify": "^10.2.4",
     "ccjs": "^0.2.0",
-    "closurecompiler": "^1.5.1"
+    "closurecompiler": "^1.5.1",
+    "node-sass": "^3.4.2"
```

Comment:
Is it possible to avoid using sass? It seems not reasonable to use it for just a couple variables, but forces css codestyle.